### ISSUE TITLE
Chrome: Bundle in the Chrome Store not available for install on non-ChromeOS machines

### ISSUE THREAD

**Issue by rubin110** [siacs/Conversations#1487]
More of wondering what the rational is behind this choice than complaining about it. :)

Generally I use a Debian laptop with my Android phone, but lately I've also been using a mix of ChromeOS, Windows 10, and (on those cold dark nights) an OS X machine. It would be nice to have a unified XMPP client between all of them.

**Comment by iNPUTmice** [comment]
It wasn't my choice. It just doesn't work. I think this might be because chrome doesn't ship with an android run time.

**Comment by rubin110** [comment]
![20151011110837-workspace 1](https://cloud.githubusercontent.com/assets/222551/10418178/7675e312-7008-11e5-8613-4c15cae141f5.png)

Hurm, I think it works for me (if I build through the ARC Welder) on non-ChromeOS because I've got the App Runtime. I built this package a while ago and am now having a hard time figuring out how I did that or where I grabbed source from. I have a feeling that was a limited time offer thing from Google. Incidentally enough I believe this project now might be helpful...

https://archon-runtime.github.io/
